airflow/jobs/scheduler_job_runner.py

```diff
@@ -721,6 +721,7 @@ def _process_executor_events(self, session: Session) -> int:
             if state == TaskInstanceState.QUEUED:
                 ti.external_executor_id = info
                 self.log.info("Setting external_id for %s to %s", ti, info)
+                session.commit()
```

Comment:
I think this is unrelated change?

Reply:
Yeah, it's part of a different PR.  Thanks for checking.

Reply:
This is fixed now, sorry for the confusion.